src/array/ops/image.rs

```diff
@@ -136,6 +130,44 @@ where
     ImageBuffer::from_raw(w, h, owned).unwrap()
 }
 
+impl<'a> From<DynamicImage> for DaftImageBuffer<'a> {
```

Comment:
Are you intending to take ownership of the buffer of DynamicImage or Borrow the buffer? Right now it's taking ownership.

Reply:
Ownership, since we should only be doing this conversion on ingest (decoding), right? I can change this to borrow the buffer if we think that will be needed.

Reply:
This is fine then, we can implement a `From<&'a DynamicImage> ` down the road when/if we need to borrow.